x-pack/plugins/ingest_manager/server/errors.ts

```diff
@@ -21,3 +21,5 @@ export const getHTTPResponseCode = (error: IngestManagerError): number => {
 };
 
 export class RegistryError extends IngestManagerError {}
+export class PackageNotFound extends IngestManagerError {}
+export class PackageOutdated extends IngestManagerError {}
```

Comment:
Would it make sense to name these two `PackageNotFoundError` and `PackageOutdatedError`, for consistency at least?